README.md

```diff
@@ -8,6 +8,16 @@ Drug names based on a Markov chain fed by this [list of drug names](https://drug
 
 ## to use:
 
+you can run any of the files containing if __name__ == '__main__', including: 
```

Comment:
thank you! 

can you:
* move this text (lines 11-20 currently) to the end of the "to use" section? 
* change the link text to the actual file names in backticks, e.g. `drug_names.py`?